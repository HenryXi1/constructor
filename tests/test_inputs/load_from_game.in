1
5 7 1 0 10 r 33 36 40 h 22 T 27 B
4 3 2 5 0 r 11 17 25 h 11 T 42 H
3 3 1 0 1 r 64 67 69 71 h 44 B 52
4 3 0 0 14 r 3 5 13 21 30 h 2 H 7 T 13 B
0 3 1 10 3 5 1 4 5 7 3 10 2 11 0 3 3 8 0 2 0 6 1 8 4 12 1 5 4 11 2 4 4 6 2 9 2 9
14
