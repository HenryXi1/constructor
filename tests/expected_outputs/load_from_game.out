                          | 0|-- 0--| 1|
                            |         |
                            1    0    2
                            |  BRICK  |
                |YH|--YR--| 3|   3  | 4|-- 4--| 5|
                  |         |         |         |
                 YR    1    6         7    2    8
                  | ENERGY  |         |  HEAT   |
      | 6|-- 9--|YT|  10  | 8|--10--| 9|   5  |10|--RR--|RT|
        |         |         |         |         |         |
       12    3   YR        14    4   15        16    5   RR
        | ENERGY  |         |  PARK   |         |  HEAT   |
      |12|   4  |YB|--18--|14|      |15|--19--|16|  10  |17|
        |         |         |         |         |         |
       20        YR    6   22        23    7   24        RR
        |         |  GLASS  |         |  BRICK  |         |
      |18|--26--|19|  11  |20|--27--|21|   3  |BT|--28--|23|
        |         |         |         |         |         |
       29    8   YR        31    9   32        BR   10   34
        |  HEAT   |         |  BRICK  |         |  BRICK  |
      |24|   8  |25|--35--|26|   2  |BB|--BR--|28|   6  |29|
        |         |         |         |         |         |
       37        38   11   39        BR   12   41        42
        |         | ENERGY  |         |  WIFI   |         |
      |30|--43--|31|   8  |32|--44--|33|  12  |34|--45--|35|
        |         |         |         |         |         |
       46   13   47        48   14   49        50   15   51
        | ENERGY  |         |  WIFI   |         |  GLASS  |
      |36|   5  |37|--52--|38|  11  |39|--53--|40|   4  |41|
        |         |         |  GEESE  |         |         |
       54        55   16   56        57   17   58        59
        |         |  WIFI   |         |  GLASS  |         |
      |RH|--60--|43|   6  |OB|--61--|45|   9  |46|--62--|47|
                  |         |         |         |
                 63        OR   18   65        66
                  |         |  GLASS  |         |
                |48|--OR--|49|   9  |50|--68--|51|
                            |         |
                           OR        70
                            |         |
                          |52|--OR--|53|
